client/lib/checkout/validation.js

```diff
@@ -25,15 +25,15 @@ import {
 export function getCreditCardFieldRules() {
 	return {
 		name: {
-			description: i18n.translate( 'Name on Card', {
-				context: 'Upgrades: Card holder name label on credit card form',
+			description: i18n.translate( 'Cardholder Name', {
+				comment: 'Cardholder name label on credit card form',
 			} ),
 			rules: [ 'required' ],
 		},
 
 		number: {
 			description: i18n.translate( 'Card Number', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Card Number', { context: 'Upgrades: Card number label on credit card form'} )` *ES Score: 15*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/card-name-labels)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).